pkgs/top-level/rust-packages.nix

```diff
@@ -21,7 +22,7 @@ let
 
 in
 
-runCommand "rustRegistry-${version}-${builtins.substring 0 7 rev}" { inherit src; } ''
```

Comment:
Do you know why the rev was sliced to the first 8 characters here? Why not put it all in there?